webpack.config.js

```diff
@@ -1,12 +1,11 @@
-const path = require('path');
-const webpack = require('webpack');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
-const HtmlWebpackPlugin = require('html-webpack-plugin');
+var path = require('path');
+var webpack = require('webpack');
+var ExtractTextPlugin = require("extract-text-webpack-plugin");
+var HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = {
     entry: {
         "bundle": "./src/vector/index.js",
-        "indexeddb-worker": "./src/vector/indexedbd-worker.js",
```

Comment:
I'm assuming this and the above is a merge gone wrong: this will break the indexeddb worker.